src/coveralls.jl

```diff
@@ -11,6 +11,7 @@ module Coveralls
     using HTTP
     using JSON
     using Compat
+    using MbedTLS
```

Comment:
this needs to be in require, don't import packages that you don't directly depend on

Reply:
Good catch, sorry